### PR TITLE
fix btc signer error in ledger mode, ref #4873

### LINKED ISSUE #4873: Ledger connected wallet error when clicking Send: "No signer"
Steps to reproduce
1. Sign out wallet
2. Connect with Ledger, choose Stacks
3. Open stacks app on Ledger device
4. Connect/Try again
5. Once connected with Stacks (not with bitcoin) click "send"
6. See error:
image

Workaround (Chrome)
7. Connect Bitcoin
8. Open bitcoin app on the ledger device
9. (click Try again if needed)
10. Try send again... no issue

A user reported that the workaround didn't work for them on Chrome nor Brave.

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/7696512018), [Test report](https://leather-wallet.github.io/playwright-reports/fix/btc-ledger)

Quick fix for #4873

### CODE REVIEW

File: src/app/components/balance/bitcoin-balance-loader.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { useNativeSegwitBalance } from '@app/query/bitcoin/balance/btc-native-segwit-balance.hooks';
+import { Signer } from '@app/store/accounts/blockchain/bitcoin/bitcoin-signer';
+
+interface BitcoinBalanceLoaderProps {
+  signer: Signer<P2Ret>;
+  children(balance: BitcoinCryptoCurrencyAssetBalance): React.ReactNode;
+}
+
+export function BitcoinBalanceLoader({ signer, children }: BitcoinBalanceLoaderProps) {
+  const currentBtcAddress = signer.address;
```
Comment (kyranjamie) [inline src/app/components/balance/bitcoin-balance-loader.tsx]: If we only need address, maybe better to pass the string only?

File: src/app/components/crypto-assets/choose-crypto-asset/crypto-asset-list.tsx [hunk 2]
```diff
@@ -2,26 +2,39 @@ import type { AllTransferableCryptoAssetBalances } from '@shared/models/crypto-a
...
-  brc20Tokens,
-}: CryptoAssetListProps) {
+export function CryptoAssetList({ cryptoAssetBalances, onItemClick }: CryptoAssetListProps) {
   const { whenWallet } = useWalletType();
 
   return (
     <CryptoAssetListLayout>
+      <BitcoinNativeSegwitAccountLoader current>
+        {signer => (
+          <BitcoinBalanceLoader signer={signer}>
```
Comment (kyranjamie) [inline src/app/components/crypto-assets/choose-crypto-asset/crypto-asset-list.tsx]: so `signer.address` here?

**Review by kyranjamie — APPROVED** [review]

File: src/app/components/balance/bitcoin-balance-loader.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/components/balance/bitcoin-balance-loader.tsx]: yeah, makes sense

**Comment by 314159265359879** [comment]
@alter-eggo tested and looking good, thanks for picking this up right away.

**Pull request merged**